packages/test-commons/src/generateIsomorphicTests.ts

```diff
@@ -32,12 +33,15 @@ function isReactClass(Component: any): boolean {
  * @param props custom props per component
  * @param children custom children per component
  * @param skipList array of component names to skip
+ * @param classNameChildList array of component names for which className is legitimately passed down to
+ * child element
```

Comment:
`array of component names where className is rendered on an arbitrary child element, rather than directly on the root element.`